engine/apps/alerts/tasks/acknowledge_reminder.py

```diff
@@ -61,6 +63,11 @@ def acknowledge_reminder_task(alert_group_pk: int, unacknowledge_process_id: str
             (alert_group.pk, unacknowledge_process_id), countdown=unacknowledge_timeout
         )
     else:
+        if alert_group.started_at < timezone.now() - relativedelta(months=1):
```

Comment:
Wondering if a month is still too much? 2 weeks is not enough? Eventually we could have some backoff strategy?
Also, maybe I would use `timezone.timedelta` instead of `relativedelta` (although I see there is no `month` support there; in any case, 1 month for `relativedelta` is a fixed amount of days, right? since you give no context of the "current" month)

Reply:
Good points, I'll pull it up into settings and make it a configurable amount of days and we can start it at 14.